static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -138,11 +138,6 @@ const enhanceSidebarToggle = (): void => {
         };
 
         checkbox.addEventListener('click', closeMenuHandler);
-
-        ophan.record({
```

Comment:
At least there is not data in the lake. Looks like you always have so and an `array` through value, instead of a string. It's on my list to look into soon(ish) and create some documentation.